packages/product-json-to-csv/src/map-product-data.js

```diff
@@ -262,16 +262,13 @@ export default class ProductMapping {
   static _mapPriceToString(price: Price): string {
     // Full price:
     // 'country-currencyCode centAmount|discounted.centAmount@customerGroup.name#channel.key$validFrom~validUntil'
+    const customerGroupName = price.customerGroup && price.customerGroup.name
```

Comment:
we can use the newer syntax here :) 

`const customerGroupName = price.customerGroup?.name`

Reply:
nice I will change this.